app/controllers/sessions_controller.rb

```diff
@@ -26,4 +26,9 @@ def update
     session[:github_token] = github_token if current_user.save
     redirect_to dashboard_path
   end
+
+  def failure
+    flash[:error] = "Connecting to Github was Unsuccessful"
+    redirect_to dashboard_path
```

Comment:
This will popup if there is an issue with github authentication (path '/auth/failure') I saw it a bunch yesterday

Reply:
Nice!